src/timesteppers.jl

```diff
@@ -277,6 +279,82 @@ function stepforward!(sol, clock, ts::FilteredRK4TimeStepper, equation, vars, pa
   return nothing
 end
 
+# ------
+# LSRK(5)4
+# ------
+"""
+    struct LSRK54TimeStepper{T} <: AbstractTimeStepper{T}
+
+A 5 stage 2 storage 4th-order Runge-Kutta timestepper for time-stepping. `∂u/∂t = F(u, t)` via:
+
+Define S² = 0 at the first stage.
+```
+for i = 1:5
+  S²   = Aᵢ*S² + dt*Fⁱ(t₀+ Cᵢdt, uⁿ);
+  uⁿ⁺¹ = uⁿ + Bᵢ*S²
+end
+```
+where
+# dt  -> time interal 
+# Aᵢ  -> A coefficients from LSRK tableau table in stage i 
+# Bᵢ  -> B coefficients from LSRK tableau table in stage i 
+# Cᵢ  -> C coefficients from LSRK tableau table in stage i 
+#  i  -> stage i 
+
+for detials, please refer to M.H. Carpenter, C.A. Kennedy, Fourth-order 2N-storage Runge–Kutta schemes, Technical Report NASA TM-109112, NASA Langley Research Center, VA, June 1994
+"""
+struct LSRK54TimeStepper{T,Vec} <: AbstractTimeStepper{T}
+    S² :: T
+    Fⁱ :: T
+    A  :: Vec
+    B  :: Vec
+    C  :: Vec
+end
+
+"""
+    LSRK54TimeStepper(equation::Equation, dev::Device=CPU())
+
+Construct a 4th-order 5 stages low storage Runge-Kutta timestepper for `equation` on device `dev`.
+"""
+function LSRK54TimeStepper(equation::Equation, dev::Device=CPU())
+  @devzeros typeof(dev) equation.T equation.dims S² Fⁱ
+  
+  T = equation.T;
+  A = T[0.0, -0.417890474499852, -1.19215169464268, -1.69778469247153, -1.51418344425716];
+  B = T[0.149659021999229, 0.379210312999627, 0.822955029386982, 0.699450455949122, 0.153057247968152];
+  C = T[ 0.0, 0.149659021999229, 0.3704009573642045, 0.6222557631344415, 0.95828213067469];
```

Comment:
Are there exact fractional values for these?

Reply:
Ya, they are the values of solution 3 from table 1 of the [paper](https://ntrs.nasa.gov/api/citations/19940028444/downloads/19940028444.pdf)

Reply:
Let's put here the rational values then?

Reply:
We could also turn those values into rational form if we want to preserve more accuracy but so it works quite well.

Reply:
Eg, for `A_2 = -567301805773//1357537059087`, etc ?

Reply:
> Eg, for `A_2 = -567301805773//1357537059087`, etc ?

Exactly this one : D
We can make it to rational value then.